src/status_im/translations/en.cljs

```diff
@@ -472,11 +472,11 @@
    :wallet-settings                      "Wallet settings"
    :wallet-manage-assets                 "Manage Assets"
    :sign-message                         "Sign Message"
-   :signing-message-phrase-description   "Sign the message by entering your password. Make sure that the words above match your secret signing phrase"
-   :signing-phrase-description           "Sign the transaction by entering your password. Make sure that the words above match your secret signing phrase"
+   :signing-message-phrase-description   "If you recognize these words, enter your login password to sign the message"
+   :signing-phrase-description           "If you recognize these words, enter your login password to sign the transaction"
    :wallet-insufficient-funds            "Insufficient funds"
    :wallet-insufficient-gas              "Not enough ETH for gas"
-   :error-unable-to-get-prices           "Currency conversion error. Try to refresh to update later"
+   :error-unable-to-get-prices           "Currency conversion error. Refresh your screen to try again."
```

Comment:
How to refresh?

Reply:
I think that depends on the OS? e.g. in Android you swipe down to refresh a screen

Reply:
I see what you mean, though we have to assume a user knows how to their device. Otherwise UI language will get stuffed with extra instruction and info

Reply:
Hmm I'm not so sure. How would a user discover it on this particular screen? If you are telling the user to refresh shouldn't you let them know how?